CHANGELOG.md

```diff
@@ -0,0 +1,139 @@
+# Release 2.0
+
+## What's new
+
+This is a brief development summary for this release.
+
+For a list of relevant Pull Requests related to the 2.0 release, see this [GitHub query result][].
+
+### Server platform and language independence
+
+-   UPWARD spec introduced for cross platform server behavior configuration.
+    An UPWARD file describes a stack-agnostic server that is used for server side rendering, asset delivery, and proxying.
+-   A PWA package can now define its network requirements in an UPWARD file.
+-   A PWA can now be deployed on any tech stack as long as the server is UPWARD compliant.
+-   UPWARD servers implemented in[NodeJS][] and [PHP][]
+-   UPWARD spec published and open for community contribution.
+
+#### Related documentation updates
+
+-   [UPWARD spec overview][]([#382][])
+-   [UPWARD-js server][]([#382][])
+
+### Improved developer experience
+
+-   Implemented consistent patterns for workflow, such as action/reducer organization and container/presentational separation, in the Venia reference theme source code.
+-   Venia components are now portable and can be used in any React application.
+-   New centralized driver and adapter concepts for all PWA-Studio component input/output.
+-   Configurable low level UI elements, such as lists and forms, available for developers without the need for drivers or adapters.
+-   Improved error handling to help detection and recoverability.
+
+#### Related documentation updates
+
+-   [Modular Venia components][] ([#796][])
+-   [Venia directory structure][]([#487][] [#752][])
+
+### App-like experience
```

Comment:
Should this title include the word "Venia" in it?

Reply:
Venia added